R/pkg/R/DataFrame.R

```diff
@@ -2973,15 +2974,51 @@ setMethod("describe",
             dataFrame(sdf)
           })
 
+#' summary
+#'
+#' Computes specified statistics for numeric and string columns.
```

Comment:
This is unchanged from before. The stats are only computed for `NumericType` and `StringType` columns. Of course the only ones that are non null for strings are count, min, and max.